test/index.js

```diff
@@ -258,6 +261,26 @@ describe('Generic', function () {
       assert(type(obj) === 'foo', 'type(obj) === "foo"');
     });
 
+    var extendedArraySupported = false;
+    var extendedArrayBody = [
+      'class ExtendedArray extends Array {',
+      '  get [Symbol.toStringTag]() {',
+      '    return "ExtendedArray";',
+      '  }',
+      '}',
+    ].join('\n');
+    try {
+      eval(extendedArrayBody); // eslint-disable-line no-eval
+      extendedArraySupported = true;
+    } catch (error) {
+      extendedArraySupported = false;
+    }
+
+    itIf(extendedArraySupported)('extended array', function () {
```

Comment:
Node 4.3 doesn't support Symbol.toStringTag so you'll need to add something like:

```js
itIf(typeof Symbol !== 'undefined' && typeof Symbol.toStringTag !== 'undefined' && extendedArraySupported)(...
```